participants/update_meta.py

```diff
@@ -151,7 +151,9 @@ def __update_meta(self, project, providers, metatype):
                                     core.http_PUT(u, data="\n".join(metadata))
                                 else:
                                     core.edit_meta(
-                                        metatype, project, data=metadata
+                                        metatype, project,
```

Comment:
The signature of core.edit_meta seems to have changed.
should project be path_args=project?
https://github.com/MeeGoIntegration/osc/blob/master/osc/core.py#L3837